lib/ui/components/scenario-overview.js

```diff
@@ -19,6 +19,12 @@ import { ScenarioRequest } from 'ui/components/scenario-request';
 
 export class ScenarioOverview extends Component {
 
+  constructor(props) {
+    super(props);
+    this.state = { masterToggle : false };
```

Comment:
Add a blank line after super, there is no need for a space between masterToggle and the : symbol